### PR TITLE
Feature/vercel fixes

### PR DESCRIPTION
**Pull request by Njoe00**
Addressing fixes for header images, scroll affects for discover tag, adding changing icons fill when using slider and changing wallet image slider to match more closely to website and have more clean landing page when initially rendering

### CODE REVIEW

File: src/components/SliderImage.tsx [hunk 1]
```diff
@@ -56,9 +56,7 @@ export default function SliderImage() {
             key={slideIndex}
             onClick={() => goToSlide(slideIndex)}
             className="text-2xl cursor-pointer"
-          >
-            <RxDotFilled />
-          </div>
+          ></div>
```
Comment (MKCMMSK) [inline src/components/SliderImage.tsx]: self closing div here would be nice

File: src/components/WalletTypeButton.tsx [hunk 2]
```diff
@@ -37,16 +37,24 @@ export default function WalletTypeButton({
...
+            isActive ? `text-orange-600` : `text-black`
+          }`}
+        >
           <BsChevronCompactDown />
         </div>
-
-        <div className="group-hover:text-orange-600 text-sm translate-y-4 text-gray-400">
+        <div
+          className={`group-hover:text-orange-600 text-sm translate-y-4 ${
+            isActive ? "text-orange-600" : "text-gray-400"
```
Comment (MKCMMSK) [inline src/components/WalletTypeButton.tsx]: Could extract isActive class and be used in both locations

File: src/pages/index.tsx [hunk 3]
```diff
@@ -187,14 +190,20 @@ export default function Home() {
...
               onClick={() => goToSlide(slideIndex)}
-              className="text-2xl cursor-pointer"
+              className="text-2xl cursor-pointer flex-row flex"
             >
-              <RxDotFilled />
+              <a
+                className={`h-3 w-3 rounded-full outline-1 outline ${
+                  currentIndex === slideIndex
+                    ? "outline-gray-400 bg-gray-400"
+                    : "outline-gray-400"
```
Comment (MKCMMSK) [inline src/pages/index.tsx]: outline-gray-400 is a common class within this className, shouldn't be in ternary

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Nits

**Comment by MKCMMSK** [comment]
image

Is this intended?

**Comment by MKCMMSK** [comment]
image

Images may still be broken 

@MKCMMSK fix is addressed in different branch.

**Comment by MKCMMSK** [comment]
image

Content is still overflowing cards. Seems only to happen when image is broken

File: src/components/SliderImage.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/SliderImage.tsx]: addressed.

File: src/components/WalletTypeButton.tsx (hunk 2 above)
Comment (Njoe00) [reply src/components/WalletTypeButton.tsx]: addressed.

File: src/pages/index.tsx (hunk 3 above)
Comment (Njoe00) [reply src/pages/index.tsx]: addressed.

**Comment by Njoe00** [comment]
> image
> 
> Images may still be broken



> image
> 
> Content is still overflowing cards. Seems only to happen when image is broken

@MKCMMSK  addrssed issue.

**Comment by Njoe00** [comment]
> image
> 
> Images may still be broken
> 
> @MKCMMSK fix is addressed in different branch.



> image
> 
> Content is still overflowing cards. Seems only to happen when image is broken
@MKCMMSK  addressed issue.

File: src/components/Header.tsx [hunk 4]
```diff
@@ -31,10 +56,19 @@ export default function Header() {
...
               <button
-                onMouseEnter={handleMouseEnter}
-                onMouseLeave={handleMouseLeave}
+                onMouseEnter={() => {
+                  handleMouseEnter();
+                  changeHeaderContent(section.text);
+                }}
+                onMouseLeave={() => {
+                  handleMouseLeave();
+                  changeHeaderContent(section.text);
```
Comment (MKCMMSK) [inline src/components/Header.tsx]: Why does this get set again on leave? That's kind of a curious implementation

File: src/components/Header.tsx [hunk 5]
```diff
@@ -31,10 +56,19 @@ export default function Header() {
...
+                }}
                 className="text-sm py-2.5 text-center inline-flex items-center hover:text-orange-600"
-                onClick={handleDropDown}
+                onClick={() => {
+                  changeHeaderContent(section.text);
+                  handleDropDown();
```
Comment (MKCMMSK) [inline src/components/Header.tsx:66]: This seems like an odd function, what does handleDropDown really do if it doesn't handle the change in content? Feels like the change in header should be moved over to handleDropDown

File: src/components/ProductCard.tsx [hunk 6]
```diff
@@ -82,8 +82,15 @@ export default function ProductCard({ content }: { content: Content }) {
...
-          <div className="top-10 relative">
-            <Image src={cardImage} height={300} width={300} alt="cardImage" />
+          <div className="pt-10 absolute">
+            <Image
+              src={cardImage}
+              height={300}
+              width={300}
+              alt="cardImage"
+              className="text-center"
+              style={{ height: 250, width: 300 }}
```
Comment (MKCMMSK) [inline src/components/ProductCard.tsx]: heights are different

Comment (MKCMMSK) [reply src/components/ProductCard.tsx]: Also, there has to be a different way to do this, seems odd you would have to declare it again in style. You mind looking into this?

File: src/pages/index.tsx [hunk 7]
```diff
@@ -198,10 +198,8 @@ export default function Home() {
               className="text-2xl cursor-pointer flex-row flex"
             >
               <a
-                className={`h-3 w-3 rounded-full outline-1 outline ${
-                  currentIndex === slideIndex
-                    ? "outline-gray-400 bg-gray-400"
-                    : "outline-gray-400"
+                className={`h-3 w-3 outline-gray-400 rounded-full outline-1 outline ${
+                  currentIndex === slideIndex ? " bg-gray-400" : "bg-white"
```
Comment (MKCMMSK) [inline src/pages/index.tsx]: Is adding bg-white intended? Wasn't there before.

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Couple more things

**Comment by MKCMMSK** [comment]
image


Default state for that button is still close. Is this intended? Or is this going to be fixed in the other PR?

File: src/components/Header.tsx (hunk 4 above)
Comment (Njoe00) [reply src/components/Header.tsx]: addressed in different branch.

File: src/components/Header.tsx (hunk 5 above)
Comment (Njoe00) [reply src/components/Header.tsx:66]: addressed in different branch.

File: src/pages/index.tsx (hunk 7 above)
Comment (Njoe00) [reply src/pages/index.tsx]: removed.

File: src/components/ProductCard.tsx (hunk 6 above)
Comment (Njoe00) [reply src/components/ProductCard.tsx]: removing styling, only using width and height properties.

File: src/components/Header.tsx [hunk 8]
```diff
@@ -60,10 +60,6 @@ export default function Header() {
                   handleMouseEnter();
                   changeHeaderContent(section.text);
                 }}
-                onMouseLeave={() => {
-                  handleMouseLeave();
-                  changeHeaderContent(section.text);
-                }}
```
Comment (MKCMMSK) [inline src/components/Header.tsx]: Why was this removed completely? I thought you want to know whether when mouse leaves button?

File: src/components/Header.tsx [hunk 9]
```diff
@@ -86,13 +82,7 @@ export default function Header() {
...
           {headerIcons.map((image, index) => (
             <span className="flex" key={index}>
-              <Image
-                src={image}
-                alt="mail"
-                width={10}
-                height={10}
-                style={{ height: "100%", width: "auto" }}
-              />
+              <Image src={image} alt="mail" width={16} height={16} />
```
Comment (MKCMMSK) [inline src/components/Header.tsx:85]: Does this fix the console warnings?

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Questions

File: src/components/Header.tsx (hunk 8 above)
Comment (Njoe00) [reply src/components/Header.tsx]: addressed in different PR.

File: src/components/Header.tsx (hunk 9 above)
Comment (Njoe00) [reply src/components/Header.tsx:85]: fixed via providing aspect ratio to className.

**Review by MKCMMSK — APPROVED** [review]
